src/entities/activity/lib/hooks/useActivitiesByIds.ts

```diff
@@ -5,10 +5,11 @@ import { useActivitiesByIdsQuery } from '../../api';
 type Props = {
   ids: Array<string>;
   enabled?: boolean;
+  isPublic: boolean;
 };
 
-export const useActivitiesByIds = ({ ids, enabled }: Props) => {
-  const queryResults = useActivitiesByIdsQuery({ ids, enabled });
+export const useActivitiesByIds = ({ ids, enabled, isPublic }: Props) => {
+  const queryResults = useActivitiesByIdsQuery({ ids, enabled, isPublic });
   ids;
```

Comment:
What is this line?

Reply:
I have noticed that this hook works in an incorrect way. There should be the isPublic param to fetch activity properly.